lib/libefi/rdwr_efi.c

```diff
@@ -1115,6 +1115,8 @@ efi_use_whole_disk(int fd)
 
 	rval = efi_alloc_and_read(fd, &efi_label);
 	if (rval < 0) {
+		if (efi_label != NULL)
+			free(efi_label);
```

Comment:
This does look like a false positive. So far results from cppcheck have been mixed, I'm not sure how much benefit would adding this tool to buildbot bring...